app/src/main/res/layout/winner_activity.xml

```diff
@@ -36,6 +36,15 @@
             android:text="Score: 11"
             android:textColor="@color/white"
             android:textSize="30sp"/>
+        <Button
+            android:id="@+id/startNew_Game_BTN"
```

Comment:
The naming convention is

<activity_name>_<BTN/IMG/LBL....>_<component_name>
In this case please change it to:

winner_BTN_startNewGame